hack/local-development/dev-setup-register-gardener

```diff
@@ -32,10 +31,16 @@ SETTINGS_APISERVICE_NAME="v1alpha1.settings.gardener.cloud"
 
 CONTROLLER_MANAGER_SERVICE_NAME="gardener-controller-manager"
 CONTROLLER_MANAGER_ENDPOINT_NAME="gardener-controller-manager"
+CONTROLLER_MANAGER_SERVICE_PORT=443
+CONTROLLER_MANAGER_PORT_STRING=""
+CONTROLLER_MANAGER_EXTERNALNAME=gardener.localhost
 
 if [[ $(k8s_env) == "$NODELESS" ]]; then
-  APISERVER_SERVICE_PORT=8443
+  APISERVER_SERVICE_PORT=$API_SERVER_SECURE_PORT
   APISERVICE_PORT_STRING="    port: $APISERVER_SERVICE_PORT"
+
+  CONTROLLER_MANAGER_SERVICE_PORT=$CONTROLLER_MANAGER_SECURE_PORT
+  CONTROLLER_MANAGER_PORT_STRING="      port: $CONTROLLER_MANAGER_SERVICE_PORT"
```

Comment:
I suspect it wouldn't hurt to set this every time (not only for a nodeless setup) and leave out line 35. WDYT?